### ISSUE TITLE
Context Menu items not executing onPressed callbacks when contextMenuBuilder parameter is supplied in QuillEditor

### ISSUE THREAD

**Issue by kaimatzu** [singerdmx/flutter-quill#1674]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/singerdmx/flutter-quill/issues)

### Flutter Quill version

9.2.2

### Steps to reproduce

I supplied the contextMenuBuilder parameter of a QuillEditor so I can create my own custom menu, the problem is all callbacks were disabled for some reason when I did this.

### Expected results

I expected all menu items to work as intended

### Actual results

Did not execute onPressed callbacks

### Code sample

<details><summary>Code sample</summary>

```dart
contextMenuBuilder: (context, state) {
  List<ContextMenuButtonItem> btnItems =
      List.from(state.contextMenuButtonItems);

  return AdaptiveTextSelectionToolbar.buttonItems(
    buttonItems: btnItems,
    anchors: state.contextMenuAnchors,
  );
},
```

</details>

I passed this as a parameter in a Quill Editor. This should just give me back all the original context menu buttons supplied by Quill. But for some reason, they don't work now. As well as other buttons that I add in the context menu.

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by yogesharora388** [comment]
Same for me. Did you find the solution ?

**Comment by kaimatzu** [comment]
> Same for me. Did you find the solution ? @kaimatzu

Nope. I think there's some issue with the context menu builder code (flutter keeps updating the context menu builder code). I think they just failed to update the code for context menus to keep up with the changes that flutter makes. Every time flutter updates stuff, they break a lot of libraries.

And I'm too busy to dive in the source code for now to find the issue myself. So I just resorted to use the toolbar. Lol
